src/listeners/thanksListener.ts

```diff
@@ -18,28 +18,28 @@ export const thanksListener: ListenerInt = {
 
       const thanksRegex =
         /((?:^|\s)(?:(?:th(?:n[qx]|x)|t[xyq]|tn(?:[x]){0,2})|\w*\s*[.,]*\s*than[kx](?:[sxz]){0,2}|than[kx](?:[sxz]){0,2}(?:[uq]|y(?:ou)?)?)|grazie|arigato(?:[u]{0,1})|doumo|gracias?|spasibo|dhanyavaad(?:hamulu)?|o?brigad(?:o|a)|dziekuje|(?:re)?merci|multumesc|shukra?an|danke)\b/gi;
-      if (!thanksRegex.test(content) || !mentions.users.size) {
+      if (!thanksRegex.test(content) || !mentions.members.size) {
```

Comment:
```suggestion
      if (!thanksRegex.test(content) || !mentions.members?.size) {
```

Unlike `users`, `members` can be undefined (as is the case in DMs, which Becca doesn't support) - using optional chaining here will invoke the early return and resolve the build error.